src/main/resources/config/en.yml

```diff
@@ -85,6 +85,8 @@ Experiment_WebhookChatMessageAvatarFromDiscord: false
 # Embed & webhook image/avatar url format
 # Alternative embed avatar url for offline servers:
 # https://cravatar.eu/helmavatar/{username}/{size}.png#{texture}
+# Use this embed avatar url if you are using GeyserMC:
```

Comment:
```suggestion
```

Reply:
Don't use this url, you can't see skins of java players in offline servers. Because uuid in java servers are generated by username. Use this url instead: https://api.creepernation.xyz/head/{username}